hermes-management/src/main/java/pl/allegro/tech/hermes/management/infrastructure/kafka/service/BrokersClusterService.java

```diff
@@ -97,6 +100,10 @@ public boolean allSubscriptionsHaveConsumersAssigned(Topic topic, List<Subscript
         }
     }
 
+    public void createConsumerGroup(Subscription subscription) {
+        consumerGroupManager.createConsumerGroup(subscription);
```

Comment:
I think this feature should be configurable (enable/disable), because there may be users that used the "earliest" offset reset policy intentionally. For them, this feature should be disabled, so consumers will behave as before.

Reply:
Done